frontend/components/movement-slip/_movement-slip.scss

```diff
@@ -51,25 +51,18 @@
     text-align: right;
     color: $govuk-secondary-text-colour;
     margin-bottom: govuk-spacing(0);
-
-    @media print {
-      font-size: 14px;
-    }
   }
 
   .movement-slip-information {
-    @extend .govuk-body;
-
-    font-weight: 300;
-    font-style: normal;
-    font-size: 14px;
-    line-height: 24px;
-    letter-spacing: 0px;
-    padding-left: 40px;
+    @extend .govuk-body-m;
+
+    display: flex;
+    align-items: center;
+    gap: 5px;
     padding-top: 10px;
+    padding-left: 7px;
     padding-bottom: 10px;
-
-    background: #F5F5F5 url("/assets/images/icon-information.svg") no-repeat 2px 50%;
+    background-color: #F5F5F5;
```

Comment:
This still needs the option to be selected in the print options to print the grey information background.  The image is included regardless.